nvim/lua/hasanhaja/plugins.lua

```diff
@@ -0,0 +1,41 @@
+return {
+  { "rose-pine/neovim",
+    name = "rose-pine",
+  },
```

Comment:
Is anything like this required? https://github.com/slydragonn/nvim-lazy/blob/main/lua/slydragonn/plugins/colorscheme.lua

Or these lazy vim opts: https://www.lazyvim.org/plugins/colorscheme

Reply:
The rose-pine colorscheme seems to be applying but the colors are wrong

Reply:
setting styles transparency to true fixed the bg color